ci/travis/build_project.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p ./build && cd ./build
+cmake -B ..
```

Comment:
the `-B` argument looks a bit weird; we don't typically use it, but if it works, fine from me